cmd/kinflate/demo/springboot.md

```diff
@@ -0,0 +1,328 @@
+# Demo: SpringBoot
+
+This example takes k8s resources for one Spring Boot application sbdemo, and customizes them to suit a
+production scenario.
+
+In the production environment we want:
+
+- add configuration for this Spring Boot application
+- customize configuration by adding prod database connection
+- resource names to be prefixed by 'prod-'.
+- resources to have 'env: prod' labels.
+- jvm memory to be properly set.
+- health check and readiness check.
+
+### Download resources
+
+Download `deployment.yaml`, `service.yaml`. These are plain k8s resources files one
+could add to a k8s cluster to run sbdemo.
+
+<!-- @makeSpringBootDir @test -->
+```
+DEMO_HOME=$(mktemp -d)
+cd $DEMO_HOME
+
+# Get SpringBoot configs
+for f in service deployment; do \
+  wget https://raw.githubusercontent.com/kinflate/example-springboot/master/$f.yaml ; \
+done
+```
+
+### Initialize a manifest
+
+A _manifest_ groups these resources together.
+
+Create one:
+
+<!-- @initApp @test -->
+```
+cd $DEMO_HOME
+kinflate init
+```
+
+You should now have a file called `Kube-manifest.yaml`:
+
+<!-- @catMan @test -->
+```
+cat $DEMO_HOME/Kube-manifest.yaml
+```
+
+containing something like:
+
+
+> ```
+> apiVersion: manifest.k8s.io/v1alpha1
+> kind: Manifest
+> metadata:
+>   name: helloworld
+> # description: helloworld does useful stuff.
```

Comment:
I am wondering if we should take out these "hello-world" stuff from the initial Kinflate configuration file ?